corehq/apps/reports/standard/users/reports.py

```diff
@@ -237,11 +262,27 @@ def _user_history_details_cell(self, changes, domain):
                 primary_changes[properties[key]] = value
                 all_changes[properties[key]] = value
         more_count = len(all_changes) - len(primary_changes)
-        return render_to_string("reports/standard/partials/user_history_changes.html", {
-            "primary_changes": self._html_list(primary_changes),
-            "all_changes": self._html_list(all_changes),
-            "more_count": more_count,
-        })
+
+        if for_export:
+            def _convert_dict_to_string(changes_dict):
+                s = ''
+                for i, key in enumerate(changes_dict):
+                    if not key:
+                        continue
+                    elif not changes_dict[key]:
+                        s += key + ": None"
+                    else:
+                        s += key + ': ' + str(changes_dict[key])
+                    if not i == len(changes_dict) - 1:
+                        s += ', \n'
+                return s
+            return _convert_dict_to_string(all_changes)
+        else:
```

Comment:
I like the clarity of `else` here because I can look at what's in the `else` statement then look at if statement right above to quickly see "oh, this is what happens when I'm not exporting". That said I haven't noticed many `else`s in HQ code so I'm not holding strongly to this by any means.

Reply:
cool, I think reasonable people can disagree on this and don't think HQ has a definitive standard